doc/kernel/other/polling.rst

```diff
@@ -172,7 +172,7 @@ to :c:macro:`K_POLL_STATE_NOT_READY` by the user.
         }
     }
 
-Using k_poll_signal()
+Using k_poll_signal_raise()
```

Comment:
The title underline on the next line must be at least as long as the title, so add some more ``===``'s

Reply:
nice catch ! thanks